plugins/ua_pki_default.c

```diff
@@ -422,48 +466,6 @@ certificateVerification_verify(void *verificationContext,
     return retval;
 }
 
-/* Find binary substring. Taken and adjusted from
- * http://tungchingkai.blogspot.com/2011/07/binary-strstr.html */
-
-static const unsigned char *
-bstrchr(const unsigned char *s, const unsigned char ch, size_t l) {
```

Comment:
Move the function bstrchr() out of the macro UA_ENABLE_ENCRYPTION_MBEDTLS. So it can be re-used.